templates/ogn-token.html

```diff
@@ -5,6 +5,49 @@
 endwith %}
 
 <main class="tokens-page">
+  <section class="exchanges-section">
+    <div class="container">
+      <div class="token-stack-image text-center my-5">
+        <img src="/static/img/ognToken/token-stack.svg" class="mx-auto" />
+      </div>
+      <h2>{{gettext("OGN are now available on these top exchanges")}}</h2>
+      <div class="row d-flex flex-column flex-md-row justify-content-around mb-4">
+        <div class="col-md-3">
+          <div class="token-exchange">
+            <img src="/static/img/ognToken/binance.svg" />
+            <a 
+              class="btn btn-outline-primary"
+              target="_blank"
+              rel="noopener noreferrer"
+              href="https://www.binance.com/en/register?ref=38914658"
```

Comment:
@joshfraser can you confirm that these are the correct referral URLs? I saw a few different Binance `ref`s in various places. 🤔